bundles/org.openhab.binding.vallox/src/main/resources/ESH-INF/thing/se-channels.xml

```diff
@@ -0,0 +1,547 @@
+
+	<channel-type id="seCo2" advanced="true">
+		<item-type>Number</item-type>
+		<label>Measured Co2</label>
+		<state min="0" max="2000" step="1" pattern="%d ppm" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2High" advanced="true">
+		<item-type>Number</item-type>
+		<label>High byte of Co2 value</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2Low" advanced="true">
+		<item-type>Number</item-type>
+		<label>Low byte of Co2 value</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2SetPoint" advanced="true">
+		<item-type>Number</item-type>
+		<label>Set point for Co2 adjustment</label>
+		<state min="0" max="2000" step="1" pattern="%d ppm" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seCo2SetPointHigh" advanced="true">
+		<item-type>Number</item-type>
+		<label>High byte of Co2 set point</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seCo2SetPointLow" advanced="true">
+		<item-type>Number</item-type>
+		<label>Low byte of Co2 set point</label>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="seAdjustmentInterval" advanced="true">
+		<item-type>Number</item-type>
+		<label>Adjustment interval</label>
+		<description>Adjustment interval of Co2 and %RH in scale of 1-15 minutes</description>
+		<state min="1" max="15" step="1" pattern="%d min" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seAutomaticHumidityLevelSeekerState" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Automatic humidity level seeker state</label>
+		<description>Automatic humidity level seeker state</description>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seBoostSwitchMode" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Boost switch mode</label>
+		<description>Boost switch mode. OFF=Fireplace, ON=Boost</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+
+	<channel-type id="seRadiatorType" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Radiator type</label>
+		<description>Radiator type. OFF=Electric, ON=Water</description>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seCascadeAdjust" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Cascade adjust</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seMaxSpeedLimitMode" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Maximum speed limit mode</label>
+		<description>When maximum fan speed is limited. OFF=with adjustments, ON=always</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+
+	<channel-type id="seServiceReminder" advanced="true">
+		<item-type>Number</item-type>
```

Comment:
Could this be a DateTime instead? Maybe you could have an event channel for when maintenance is required?

Reply:
**Changed this name to `ServiceReminderMonths` for clarity.**

This channel sets the month count between maintenance (maintenance interval). Number of weeks or days can't be retrieved from the machine so nothing precise can't be calculated and setting a DateTime imho isn't feasible. 

> Maybe you could have an event channel for when maintenance is required?

 `serviceReminderIndicator` indicates when maintenance is required.